app/controllers/application_controller.rb

```diff
@@ -3,7 +3,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_current_user
-  before_action :access_control
+  before_action :restrict_to_employee
   before_action :check_primary_account
```

Comment:
Minor optimization: order might be better as

```ruby
before_action :check_primary_account
before_action :set_current_user
before_action :restrict_to_employee
```

No need for a database lookup if we aren't going to let them in anyways.